xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -1549,6 +1549,9 @@ core.recyclebin.confirm=Are you sure you wish to move this page to the recycle b
 core.recyclebin.confirmWithInlinks=Are you sure you wish to move this page to the recycle bin?
 core.recyclebin.completelyDeleteConfirm=This action is not reversible. Are you sure you wish to continue?
 core.recyclebin.invalidEntry=Invalid recycle bin entry.
+core.recyclebin.wantToSkip=Are you sure you wish to delete this page?
+core.recyclebin.wantToSkip.no=Delete and move to the recycle bin.
+core.recyclebin.wantToSkip.yes=Permanently delete the page (it won't be put in the recycle bin).
```

Comment:
It doesn't seem to follow the L10N guidelines: See https://dev.xwiki.org/xwiki/bin/view/Community/DevelopmentPractices#HTranslationBestPractices which leads to  https://dev.xwiki.org/xwiki/bin/view/Community/L10N%20Conventions . This looks like hints to display no?